subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/UnknownIncrementalAnnotationProcessingIntegrationTest.groovy

```diff
@@ -93,6 +93,6 @@ class UnknownIncrementalAnnotationProcessingIntegrationTest extends AbstractIncr
         outputs.deletedClasses("AThing")
 
         and:
-        output.contains("Annotation processor path has been changed")
+        output.contains("Input property 'options.annotationProcessorPath' file ")
```

Comment:
It would help me understand this test easier if I could see more of this message. Can you use `matchesRegex()` or something to capture more of the context here?

Reply:
Now the full message is included.

Reply:
BTW, we may want to log something special when a non-incremental input property changed for an incremental task. WDYT?

Reply:
Do you mean that we should include whether or not the property with the changes was incremental?

Reply:
That sounds like a good idea to me. At least for incremental tasks. WDYT?

Reply:
I'd do it for any task. Maybe we should add a warning for an incremental property on a non-incremental task? We should probably also distinguish between explicitly `@Incremental` properties and implicit ones using `@SkipWhenEmpty`. The latter should be allowed on non-incremental tasks, too. We could do these as a separate PR, though.